File: src/C#/StructuralPatterns/Composite/WindowsFormsApplication/Leaves/TextBox.cs
using System.Drawing;
using WindowsFormsApplication.Leaves.Abstraction;

namespace WindowsFormsApplication.Leaves
{
    public class TextBox : BaseControlLeaf
    {
        public TextBox(string name, int width, int height, Color foreColor, Color backColor)
            : base(name, width, height, foreColor, backColor)
        {

        }
    }
}

File: src/PHP/StructuralPatterns/Composite/WindowsFormsApplication/Leaves/TextBox.php
<?php

namespace StructuralPatterns\Composite\WindowsFormsApplication\Leaves;

class TextBox extends BaseControlLeaf
{
    public function __construct($name, $width, $height, $foreColor, $backColor)
    {
        parent::__construct($name, $width, $height, $foreColor, $backColor);
    }
}